### PR TITLE
Gif support for pxBenchmark

### PR DESCRIPTION
**Pull request by ashwinikalokhe**

### CODE REVIEW

File: examples/pxBenchmark/src/CMakeLists.txt [hunk 1]
```diff
@@ -57,14 +57,27 @@ set(PXSCENE_DEFINITIONS )
 set(PXSCENE_ADDITIONAL_RESOURCES )
 set(PXSCENE_INSTALLER 0)
 
+if (BUILD_WITH_GIF)
+message("Building with GIF support")
+set(GIF ${EXTDIR}/giflib-5.1.9/)
```
Comment (mfiess) [inline examples/pxBenchmark/src/CMakeLists.txt]: Please add a PREFER_SYSTEM_LIBRARIES check to see if we should be using the system version or the version of GIF from externals.

Comment (ashwinikalokhe) [reply examples/pxBenchmark/src/CMakeLists.txt]: we have that check in pxCore/src/Cmakelist.txt .. Do you want it here too? We dont have that in pxScene2d/src/CmakeLists.txt.

Comment (mfiess) [reply examples/pxBenchmark/src/CMakeLists.txt]: It looks like GIF is used to the set link directories for GIF.  However, this should only be set to the external location (external/giflib-5.1.9) if we're not using system libraries.

Comment (ashwinikalokhe) [reply examples/pxBenchmark/src/CMakeLists.txt]: Yes, we always link to external location (external/giflib-5.1.9). in pxCore/src/CmakeLists.txt we specify or link either to systems libraries or externals based on flag...

if (BUILD_WITH_GIF)
message("Building with GIF support")
if(NOT PREFER_SYSTEM_LIBRARIES)
include_directories(AFTER ${EXTDIR}/giflib-5.1.9/)
endif(NOT PREFER_SYSTEM_LIBRARIES)
add_definitions(-DSUPPORT_GIF)

endif (BUILD_WITH_GIF)

**Pull request merged**